cpg_infra/config/config.py

```diff
@@ -123,6 +123,10 @@ class GCP(DeserializableDataclass):
         gcp: GCP
         slack_channel: str | None = None
         etl_accessors: list[str] = dataclasses.field(default_factory=list)
+        etl_environment: str | None = None
+        etl_default_config: dict[str, str] | None = None
+        etl_private_repo_url: str | None = None
```

Comment:
The private_repo_url you should be able to pull directly from the resource created in #152 (I hope)

```suggestion
```

Reply:
Good point, I had removed that entry.